x-pack/solutions/security/plugins/security_solution/server/endpoint/routes/resolver/tree/queries/stats.ts

```diff
@@ -56,7 +56,7 @@ export class StatsQuery extends BaseResolverQuery {
               ? [{ term: { 'agent.id': this.agentId } }]
               : []),
             {
-              term: { 'event.kind': 'event' },
+              terms: { 'event.kind': ['event', 'alert', 'signal'] },
```

Comment:
I don't think we want to do this, alerts/signals are intentionally kept separate from the underlying process data